src/konnektor/network_planners/generators/explicit_network_generator.py

```diff
@@ -87,14 +85,13 @@ def generate_network_from_indices(
 
         Parameters
         ----------
-        components : list of Components
-          the small molecules to place into the network
-        mapper: AtomMapper
-          the atom mapper to use to construct edges
-        indices : list of tuples of indices
-          the edges to form where the values refer to names of the small molecules,
-          eg `[(3, 4), ...]` will create an edge between the 3rd and 4th molecules
-          remembering that Python uses 0-based indexing
+        components : list[Component]
+            ``Component``/s to place into the network.
+
+        indices : list[tuple[int, int]]
+            Edges to form between the components, represented as tuples of indices of the list of components.
```

Comment:
```suggestion
            Edges to form between the ``Components``, represented as tuples of indices of the list of ``Component``/s.
```

Reply:
You marked this as resolved without either saying "nah" or making a change, so unclear on status of this one.

Note here that Components imho makes more sense than components (the kwarg) because it's directly following the `Components` definition above. Unlike in the Raises section below.

Reply:
Oh I'm sorry, I thought I had applied this suggestion! I agree, and it should be committed now.